README.md

```diff
@@ -153,3 +153,56 @@ class MyComponent extends Component {
 }
 ```
 
+## Testing using React Test Utilities
+
+[lib/react_test_utils.dart](lib/react_test_utils.dart) is a Dart wrapper for the [React TestUtils](http://facebook.github.io/react/docs/test-utils.html) library allowing for tests to be made for React components in Dart.
+
+Here is an example of how to use React TestUtils within a Dart test.
+
+```dart
+import 'package:unittest/unittest.dart';
+import 'package:react/react.dart' as react;
+import 'package:react/react_client.dart' as reactClient;
+import 'package:react/react_test_utils.dart' as reactTestUtils;
+
+class MyTestComponent extends react.Component {
+  getInitialState() => {'text': 'testing...'};
+  render() {
+    return react.div({}, [
+        react.button({'onClick': (e) => setState({'text': 'success'})}),
+        react.span({'className': 'spanText'}, state['text'])
+    ]);
+  }
+}
+
+var myTestComponent = react.registerComponent(() => new MyTestComponent());
+
+void main() {
+  reactClient.setClientConfiguration();
+  var component = reactTestUtils.renderIntoDocument(myTestComponent({}));
```

Comment:
I'd like to move this component initialization either into a setup method or the test itself.  While this works for a single test in an example it can lead to state pollution when writing multiple tests and I'd like the example to not lead people into a trap when they copy/paste it.